services/app-api/utils/sanitize/sanitize.test.ts

```diff
@@ -1,58 +1,141 @@
 import { sanitizeArray, sanitizeObject, sanitizeString } from "./sanitize";
-const string = '<img id="img1" src="foo.png" onload="alert("Hello!")" />';
-const object = {
-  string,
-  numbers: 2349872,
-  array: [
-    234234,
-    "test",
-    '<math><mi//xlink:href="data:x,<script>alert(4)</script>">',
-    "<svg><g/onload=alert(2)//<p>",
-  ],
-  object: {
-    string,
-    numbers: 2349872,
-    array: [
-      "<UL><li><A HREF=//google.com>click</UL>",
-      "test2",
-      234234,
-      "<svg><g/onload=alert(2)//<p>",
-    ],
-    array2: [],
-  },
+
+// SAFE TYPES
+
+const safeBoolean = true;
+// const safeNaN = NaN;
```

Comment:
Do we want to remove this commented line or add a `TODO` for later?

Reply:
Added it back in. Must have accidentally commented it out. 🤦